create.html

```diff
@@ -37,7 +37,6 @@ <h2 class="mdl-card__title-text">Create a new DoWhop</h2>
                   <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
```

Comment:
I can't remember the precisely how the create html is used. I think Rae may be the person to discuss this with initially.